libmproxy/console/flowlist.py

```diff
@@ -136,6 +138,17 @@ def server_replay_prompt(self, k):
                 self.state.last_saveload,
                 self.master.server_playback_path
             )
+    def server_copy_response(self, k):
+        if k == "c":
+            try:
+                pyperclip.copy(self.flow.response_content())
+            except TypeError:
+                self.master.statusbar.message("Content is binary or can be converted to text")
```

Comment:
+ not :-)
We should indicate here that the content can be dumped to disk by pressing w (?) alternatively.